src/DSFR.res

```diff
@@ -1,3 +1,5 @@
+// TODO: clean the dsfr bindings + publish as a standalone package
```

Comment:
Yes publishing as a standalone package would be nice, you could even have it hosted by @codegouvfr ?

Reply:
I don't know, @bzg?